com.unity.netcode.gameobjects/Tests/Runtime/NetworkTransform/NetworkTransformOwnershipTests.cs

```diff
@@ -256,10 +256,10 @@ public override void OnNetworkSpawn()
 
             protected override void Update()
             {
-                CanCommitToTransform = IsOwner;
                 base.Update();
-                if (NetworkManager.Singleton != null && (NetworkManager.Singleton.IsConnectedClient || NetworkManager.Singleton.IsListening))
+                if (IsSpawned)
```

Comment:
It felt like this test was checking `IsOwner` at inappropriate times. This is my attempt at making it check it at correct times only..